app/js/app_routes.js

```
@@ -1,81 +1,114 @@
-
-                    console.log("securize: not authorized");
-
-                    $location.search({ path: $location.url() });
-                    $location.path(appConfigService.getSiteMapUrls().errors.notAuthorized);
-                }
-
-                return user;
-            });
-        }];
-    }
+import app from '~/app';
+import _ from 'lodash';
+import '~/services/app-config-service';
+import '~/components/scbd-angularjs-services/main';
+
+import { securize, mapView, importQ, injectRouteParams } from './mixin';
+import * as angularViewWrapper from '~/views/shared/angular-view-wrapper'
+
+var baseUrl = require.toUrl("").replace(/\?v=.*$/, "");
+
+
+const routeTemplates = {
+
+    views_home_index          : { component: ()=>import('~/views/home/index') },
+    views_projects_index      : { component: ()=>import('~/views/projects/index') },
+    views_translation_index   : { component: ()=>import('~/views/translation/index') },
+    views_translation_project : { component: ()=>import('~/views/translation/project') },
+    views_translation_database: { component: ()=>import('~/views/translation/database-tables') },
+    views_translation_table   : { component: ()=>import('~/views/translation/table') },
+    views_article_index       : { component: ()=>import('~/views/article/index') },
+    views_article_editor      : { component: ()=>import('~/views/article/editor') },
+    views_article_view        : { component: ()=>import('~/views/article/view') },
+    views_workflows_index     : { component: ()=>import('~/views/workflows/index') },
+    views_widgets_index       : { component: ()=>import('~/views/widgets/index') },
+    views_widgets_edit        : { component: ()=>import('~/views/widgets/edit') },
+    views_widgets_view        : { component: ()=>import('~/views/widgets/view') },
+    views_tags_index          : { component: ()=>import('~/views/tags/index') },
+    views_shared_login        : { component: ()=>import('~/views/shared/login') },
```

should add somthing like `import(...).catch(logError)` to avoid silent errors or  `tryImport()` function
eg:
```javascript
function logError(err) {
    console.log(err)
    throw err;
}
```

---

done